### DISCUSSION TITLE
Waiting for Weltpixel Google Analytics 4 (GA4) With GTM Support for Magento 2 Integrations with Breeze

### DISCUSSION THREAD

**Discussion by Mizogorou** [breezefront/community#195] [3. Integrations]
Hi there,

I bought Pro version of Weltpixel Google Analytics 4 with GTM (https://www.weltpixel.com/google-analytics-4-ga4-with-gtm-support-for-magento-2.html) but it doesn't work with Breeze now.

Is it possible to make an integrartion for this? 😘😘

Thanks in advance.

**Comment by dimdimich** [comment]
Please simply install https://github.com/breezefront/module-breeze-weltpixel-ga4

**Reply by Mizogorou** [reply]
Wow, Thank you.

**Reply by Mizogorou** [reply]
Hi @dimdimich,

It doesn't work. Just an only scroll tag fired.
<img width="1680" alt="Screenshot 2566-10-05 at 02 33 43" src="https://github.com/breezefront/community/assets/8059336/fd5e7f69-2802-4ea2-908a-9fbf00766089">

**Reply by Mizogorou** [reply]
Hi @dimdimich ,

I am sorry for previous reply, above debug result is invalid because my misconfiguration (We have to set "Automatically disable data layer for enabled measurement protocol events" option to "No" on debug mode).

Thanks for your help.

**Reply by sandipansaha** [reply]
Hi @dimdimich Can we please update the document if it is officially supported and maintained - https://breezefront.com/extensions ?

**Comment by dimdimich** [comment]
Thank you! We forgot to add it to this list. We just added it.
